rhba-dev-env/ansible/group_vars/all/vars_file.yml

```diff
@@ -0,0 +1,59 @@
+---
+### environment vars ###
+workspace_home:  /home/vagrant/workspace
+# https://github.com/ansible/ansible/issues/54855
+ansible_python_interpreter: /usr/bin/python3
+# same "ip" from config.yml (Vagrantfile on vagrant-ocp-bsig)
+vm_ip: 192.168.0.40
+# same "name" from config.yml (Vagrantfile)
+vm_name: workstation
+
+### kerberos config ###
+kerberos_username: mmagnani
```

Comment:
The default should not be your username. Perhaps "my_kerberos_username", similar to "my_kerberos_password"?

Reply:
yep, done!